boxfile.yml

```diff
@@ -43,7 +47,7 @@ data.minio_storage:
 
 web.main:
   start:
-    nginx: nginx -c /app/contentcuration/etc/nginx.conf
+    nginx: nginx -c /app/deploy/nginx.conf
     django: gunicorn -c etc/gunicorn.py contentcuration.wsgi:application
```

Comment:
I'm wondering if we can also put `gunicorn.py` into `/app/deploy/` instead of `etc/`?

Reply:
@lyw07 yeah! I can transfer that over.